packages/types/src/items.d.ts

```diff
@@ -0,0 +1,18 @@
+export type ItemStatus = "available" | "reserved" | "in-use" | "unavailable"; // 예약가능, 예약됨, 대여중, 사용불가
+
+export type ItemType = "seat" | "room" | "equipment";
+
+export interface TItem {
```

Comment:
```suggestion
export interface IItem {
```
interface는 I부착...? 좀 웃기긴 하네 ㅑㅑtem